inc/saihostintf.h

```diff
@@ -594,28 +497,33 @@ typedef enum _sai_hostif_attr_t
     SAI_HOSTIF_ATTR_TYPE = SAI_HOSTIF_ATTR_START,
 
     /**
-     * @brief Assosiated port or router interface
+     * @brief Host interface object ID
      *
-     * Valid only when #SAI_HOSTIF_ATTR_TYPE == #SAI_HOSTIF_TYPE_NETDEV
+     * Valid only when #SAI_HOSTIF_ATTR_TYPE == #SAI_HOSTIF_TYPE_PORT 
+     * || #SAI_HOSTIF_TYPE_LAG || #SAI_HOSTIF_TYPE_VLAN
+     * should be port object when type is SAI_HOSTIF_TYPE_PORT
+     * should be lag object when type is SAI_HOSTIF_TYPE_LAG
+     * should be VLAN rif object when type is SAI_HOSTIF_TYPE_VLAN
      *
      * @type sai_object_id_t
-     * @objects SAI_OBJECT_TYPE_PORT, SAI_OBJECT_TYPE_ROUTER_INTERFACE
+     * @objects SAI_OBJECT_TYPE_PORT, SAI_OBJECT_TYPE_LAG, SAI_OBJECT_TYPE_ROUTER_INTERFACE
```

Comment:
can we still use SAI_HOSTIF_TYPE_NETDEV here? Since we can use the object id to know whether it is port-based, lag-based, or router interface based host interface type.